Tests/StoreKitUnitTests/BeginRefundRequestHelperTests.swift

```diff
@@ -60,37 +62,6 @@ class BeginRefundRequestHelperTests: TestCase {
         }
     }
 
-    func testBeginRefundRequestForProductFatalErrorIfNotIosOrCatalyst() throws {
-        #if os(watchOS) || os(macOS) || os(tvOS)
-        expectFatalError(expectedMessage: Strings.purchase.begin_refund_request_unsupported.description) {
-            _ = try await helper.beginRefundRequest(forProduct: mockProductID)
-        }
-        #else
-        throw XCTSkip("beginRefundRequestForProduct is not available for this test.")
-        #endif
-    }
-
-    func testBeginRefundRequestForEntitlementFatalErrorIfNotIosOrCatalyst() throws {
-        #if os(watchOS) || os(macOS) || os(tvOS)
-        expectFatalError(expectedMessage: Strings.purchase.begin_refund_request_unsupported.description) {
-            _ = try await helper.beginRefundRequest(forEntitlement: mockEntitlementID)
-        }
-        #else
-        throw XCTSkip("beginRefundRequestForEntitlement is not available for this test.")
-        #endif
-    }
-
-    func testBeginRefundRequestForActiveEntitlementFatalErrorIfNotIosOrCatalyst() throws {
-        #if os(watchOS) || os(macOS) || os(tvOS)
-        expectFatalError(expectedMessage: Strings.purchase.begin_refund_request_unsupported.description) {
-            _ = try await helper.beginRefundRequestforActiveEntitlement()
-        }
-        #else
-        throw XCTSkip("beginRefundRequestForEntitlement is not available for this test.")
-        #endif
-    }
-
-#if os(iOS)
```

Comment:
Why are these being deleted? 🤔

Reply:
Oh I forgot to explain. The expectation wasn't being met (maybe it was before?). The methods aren't even being compiled anymore.

Reply:
Literally no code used that message string